mmselfsup/models/heads/mae_linprobe_head.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.cnn.utils.weight_init import trunc_normal_
+from mmcv.runner import BaseModule
+from torch import nn
+
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class MAELinprobeHead(BaseModule):
+    """Linear probing head for MAE.
+
+    Args:
+        embed_dim (int): The dim of the feature before the classifier head.
+        num_classes (int): The total classes. Defaults to 1000.
+    """
+
+    def __init__(self, embed_dim, num_classes=1000):
+        super(MAELinprobeHead, self).__init__()
+        self.head = nn.Linear(embed_dim, num_classes)
+        self.bn = nn.BatchNorm1d(embed_dim, affine=False, eps=1e-6)
+        self.criterion = nn.CrossEntropyLoss()
+        nn.init.constant_(self.head.bias, 0)
```

Comment:
should do that in init_weights()

Reply:
Thank you for your valuable comments. I will fix it as soon as possible.